app/models/assessment_detail.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class AssessmentDetail < ApplicationRecord
+  belongs_to :planning_application
+  belongs_to :user
+
+  enum status: {
```

Comment:
Is it worth having a default status?

Reply:
Hmm not sure, I don't think so
What would be the use case for this ?

Reply:
I guess I was assuming that it's 'in progress' from the moment it exists...but then perhaps it's better to set that explicitly... 🤔

Reply:
It could be `complete` too as the record persists if the officer chooses to select "save and mark as complete"